init_templates/gemspec.txt

```diff
@@ -0,0 +1,32 @@
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'openstudio/GEM_NAME_UNDERSCORES/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'GEM_NAME_BARE'
```

Comment:
if this is an extension using modules, then should be hyphened, otherwise, underscores are okay. Not sure how to document this.

Reply:
What is an extension using modules?  A gem?